hermes/probe/probe.go

```diff
@@ -51,19 +51,33 @@ type Target struct {
 	LatencyMetrics *metrics.Metrics
 }
 
-// MonitorProbe holds aggregate information about all probe runs, per-target.
+// Probe holds aggregate information about all probe runs, per-target.
 // It also holds the config and options used to initialise the probe.
-type MonitorProbe struct {
+type Probe struct {
 	name    string
 	config  *probepb.HermesProbeDef
 	targets []*Target
 	opts    *options.Options
 	logger  *logger.Logger
 }
 
+// getIntervalSec returns the probing interval in seconds as a time.Duration.
```

Comment:
Avoid the "get" prefix in general. How about "probeInterval" or just "interval".
Also, the "Sec" in the name is misleading, as it returns time.Duration not seconds.

Reply:
Okay, thanks, I've amended these.